### ISSUE TITLE
уведомление пользователя о нехватке ресурсов

### ISSUE THREAD

**Issue by edejin** [yiotro/Achipato#1]
в пятой миссии если сначала построить ферму и мясорубку с шахтой, а только потом фабрику, то запасов ресурсов (выпускаемых самой фабрикой) уже просто не хватает. Видимо надо как-то уведомить пользователя, что строить не из чего.

**Comment by yiotro** [comment]
Да, я думаю что-то делать с этим в будущих обновлениях. Кстати, я так понимаю что комментарий по поводу achikaps, а тут выложена achipato, это другая игра :)

**Comment by edejin** [comment]
Эк. Простите.

**Comment by edejin** [comment]
Создайте хоть пустой репо для achikaps, мне есть еще что сказать :)
И это не только баги, а и то что вы молодец.

**Comment by yiotro** [comment]
Лучше пишите мне на почту, я её проверяю каждый день
